sdk/template/template/package.json

```diff
@@ -61,6 +61,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/core-http": "^1.0.0",
+    "@opentelemetry/types": "^0.2.0",
```

Comment:
@mikeharder I had to add this package as a direct dependency because rush didn't put `@opentelemetry/types` into the local `node_modules` folder, even though `@opentelemetry/types` is a direct dependency of `@azure-core-http` and we depend on that. @bterlson thought you might have insights?

Reply:
I am not able to repro this. I checked out this PR, removed the package.json dependency, did a rush update, confirmed that @opentelemetry/types is not present in the local node_modules folder, and things worked fine. Can you try removing this and see if the problem still repros?

Reply:
Also FWIW, only direct dependencies should be present in node_modules. Transitive dependencies will be looked up relative to the dependent's position in the common module store.

Reply:
I tried this:

1. Remove `@opentelemetry/types` from template's `package.json`
2. Run `rush update`
3. Verify that `template\node_modules` no longer had an `@opentelemetry` folder
4. Run `rush rebuilt -t template`

The last step failed with

```
dist-esm/src/index.js → browser/azure-template.js...
[!] Error: 'CanonicalCode' is not exported by ..\..\..\common\temp\node_modules\.registry.npmjs.org\@opentelemetry\types\0.2.0\node_modules\@opentelemetry\types\build\src\index.js
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
..\..\core\core-tracing\dist-esm\lib\tracers\test\testSpan.js (5:9)
3: // Licensed under the MIT License.
4: import { NoOpSpan } from "../noop/noOpSpan";
5: import { CanonicalCode } from "@opentelemetry/types";
            ^
6: /**
7:  * A mock span useful for testing.
Error: 'CanonicalCode' is not exported by ..\..\..\common\temp\node_modules\.registry.npmjs.org\@opentelemetry\types\0.2.0\node_modules\@opentelemetry\types\build\src\index.js
    at error (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:9427:30)
    at Module.error (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:13410:9)
    at handleMissingExport (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:13330:21)
    at Module.traceVariable (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:13715:17)
    at ModuleScope.findVariable (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:12422:39)
    at FunctionScope.findVariable (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:2981:38)
    at ChildScope.findVariable (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:2981:38)
    at FunctionScope.findVariable (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:2981:38)
    at ChildScope.findVariable (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:2981:38)
    at MemberExpression.bind (C:\src\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\rollup\1.20.1\node_modules\rollup\dist\rollup.js:11337:49)
================================

[@azure/template] Returned error code: 1
```

Did you rebuild `core-tracing` and `core-http` with the changes?

Reply:
Guess I failed to build core-http. With that built, the problem repros. Filed this issue: https://github.com/rollup/rollup-plugin-commonjs/issues/419.

I believe moving to a relative path (something like `../../../common/temp/node_modules/.registry.npmjs.org/@opentelemetry/types/build/src/index.js`) should work and is likely preferable to adding a needless direct dep.

Reply:
I tried the relative path thing and I couldn't get it to work. It didn't work with your example path and it didn't work given the exact path specified in the error message, with either forward or backslashes.